src/main/server.ts

```diff
@@ -9,6 +10,12 @@ const { Logger } = require('@hmcts/nodejs-logging');
 
 const logger = Logger.getLogger('server');
 
+let httpServer: http.Server | null = null;
```

Comment:
This can be removed if the regression on line 33 is fixed as it will always be of HTTPS type.